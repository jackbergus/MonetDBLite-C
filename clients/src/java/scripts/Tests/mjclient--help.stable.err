stderr of test 'mjclient--help` in directory 'src/java/scripts` itself:


# 16:36:12 >  
# 16:36:12 >  Mtimeout -timeout 300 ./mjclient--help mjclient--help 
# 16:36:12 >  


# 07:35:51 >  
# 07:35:51 >  Mtimeout -timeout 60 mjclient --help
# 07:35:51 >  

= Please remove this line in case mjclient's usage message is supposed to go to stdout instead of stderr (as usually).

# 15:14:07 >  
# 15:14:07 >  Done.
# 15:14:07 >  

