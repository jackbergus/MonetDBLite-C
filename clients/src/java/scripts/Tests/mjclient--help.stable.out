stdout of test 'mjclient--help` in directory 'src/java/scripts` itself:


# 16:36:12 >  
# 16:36:12 >  Mtimeout -timeout 300 ./mjclient--help mjclient--help 
# 16:36:12 >  


# 07:35:51 >  
# 07:35:51 >  Mtimeout -timeout 60 mjclient --help
# 07:35:51 >  

= Please remove this line in case the below usage message of mjclient is correct.
Usage java -jar jdbcclient.jar
                  [-h host[:port]] [-p port] [-f file] [-u user]
                  [-l language] [-b database] [-e] [-d [table]]
                  [-X<opt>]
or using long option equivalents --host --port --file --user --language
--dump --echo --database.
Arguments may be written directly after the option like -p50000.

If no host and port are given, localhost and 50000 are assumed.  An .monetdb
file may exist in the user's home directory.  This file can contain
preferences to use each time JdbcClient is started.  Options given on the
command line override the preferences file.  The .monetdb file syntax is
<option>=<value> where option is one of the options host, port, file, mode
debug, or password.  Note that the last one is perilous and therefore not
available as command line option.
If no input file is given using the -f flag, an interactive session is
started on the terminal.

OPTIONS
-h --host     The hostname of the host that runs the MonetDB database.  A port
              number can be supplied by use of a colon, i.e. -h somehost:12345.
-p --port     The port number to connect to.
-f --file     A file name to use either for reading or writing.  The file will
              be used for writing when dump mode is used (-d --dump).  In read
              mode, the file can also be an URL pointing to a plain text file
              that is optionally gzip compressed.
-u --user     The username to use when connecting to the database.
-d --database Try to connect to the given database (only makes sense if
              connecting to a DatabasePool, M5 or equivalent process).
-l --language Use the given language, for example 'xquery'.
--help        This help screen.
--version     Display driver version and exit.
-e --echo     Also outputs the contents of the input file, if any.
-q --quiet    Suppress printing the welcome header.
-D --dump     Dumps the given table(s), or the complete database if none given.
-Xblksize     Specifies the blocksize when using block mode, given in bytes.
-Xoutput      The output mode when dumping.  Default is sql, xml may be used for
              an experimental XML output.
-Xembedded    Uses an "embedded" Mserver instance.  The argument to this option
              should be in the form of path/to/mserver:dbname[:dbfarm[:dbinit]].
-Xhash        Use the given hash algorithm during challenge response.  Supported
              algorithm names: SHA1, MD5, plain.
-Xdebug       Writes a transmission log to disk for debugging purposes.  If a
              file name is given, it is used, otherwise a file called
              monet<timestamp>.log is created.  A given file never be
              overwritten; instead a unique variation of the file is used.
-Xbatching    Indicates that a batch should be used instead of direct
              communication with the server for each statement.  If a number is
              given, it is used as batch size.  i.e. 8000 would execute the
              contents on the batch after each 8000 statements read.  Batching
              can greatly speedup the process of restoring a database dump.
= Please remove this line in case the above usage message of mjclient is correct.

# 09:43:26 >  
# 09:43:26 >  Done.
# 09:43:26 >  

